gui/wxpython/datacatalog/tree.py

```diff
@@ -312,10 +322,27 @@ def _reloadLocationNode(self, location_node):
                 None))
         p.start()
         maps, error = q.get()
+
         for mapset in maps:
+            mapset_path = os.path.join(location_node.parent.data['name'],
```

Comment:
This all need to be called for all reload functions, not just that one, look eg. what reloadtreenodes is calling

Reply:
I edited the way how to append mapset node in functions _reloadLocationNode, _reloadGrassDBNode and GetCurrentDbLocationMapsetNode. Also after these edits I have still the same above-mentioned "key" error. I think I should define "lock" and "user" keys maybe in treemodel? But it seems strange to me because treemodel is very general function and there is no explicit definition of what keys and values actually are.